src/content/concepts/index.md

```diff
@@ -65,7 +65,7 @@ module.exports = {
 };
 ```
 
-In the example above, we use the `output.filename` and the `output.path` properties to tell webpack the name of our bundle and where we want it to be emitted to.
+In the example above, we use the `output.filename` and the `output.path` properties to tell webpack the name of our bundle and where we want it to be emitted to. In case you you're wondering about the path module being imported at the top, it is a core NodeJS module that gets used to manipulate file paths.
 
```

Comment:
You can also add a link to core [Node.js modules](https://nodejs.org/api/modules.html).
Also, fix the typo here "In case ~you~ you're" and ~NodeJS~ => Node.js